apps/condo/domains/common/constants/requisites.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-    SUPPORT_EMAIL: 'info@doma.ai',
+    SUPPORT_EMAIL: 'help@doma.ai',
     BOT_EMAIL: 'service@doma.ai',
-    SUPPORT_PHONE: '8 (800) 923-23-23',
-}
\ No newline at end of file
+    SUPPORT_PHONE: '+7 (922) 180-75-26',
```

Comment:
Please don't use ( and ) in phone number. They were used to indicate the optional part of the number, but now this is not true

You can check this article as a reference https://tema.livejournal.com/3068951.html?page=3